fineract-provider/src/main/java/org/apache/fineract/batch/command/CommandStrategyProvider.java

```diff
@@ -128,7 +128,21 @@ private static void init() {
         commandStrategies.put(
                 CommandContext.resource("loans\\/external-id\\/" + UUID_PARAM_REGEX + OPTIONAL_QUERY_PARAM_REGEX).method(GET).build(),
                 "getLoanByExternalIdCommandStrategy");
+        commandStrategies.put(CommandContext.resource("savingsaccounts\\/" + NUMBER_REGEX + OPTIONAL_QUERY_PARAM_REGEX).method(GET).build(),
```

Comment:
You can use a generic command strategy to cover all these scenarios like:
`SavingsAccountTransactionCommandStrategy` and use 
```
        final Map<String, String> queryParameters = CommandStrategyUtils.getQueryParameters(relativeUrl);
        final String command = queryParameters.get("command");
```
to get the command query param

Reply:
fixed